examples/pipeline_wavernn/loss_mol.py

```diff
@@ -0,0 +1,81 @@
+import torch
+import torch.nn.functional as F
+
+# Adapted from wavenet vocoder:
+# https://github.com/r9y9/wavenet_vocoder/blob/master/wavenet_vocoder/mixture.py
+# Explain mol loss:
+# https://github.com/Rayhane-mamah/Tacotron-2/issues/155
+
+# Remove numpy dependency
```

Comment:
lint?

Reply:
This part has been moved to the docstring.